extensions/splunk/deployment/src/main/java/org/apache/camel/quarkus/component/splunk/deployment/SplunkProcessor.java

```diff
@@ -67,9 +67,10 @@ ReflectiveClassBuildItem registerForReflection(CombinedIndexBuildItem combinedIn
 
     @BuildStep
     List<ReflectiveClassBuildItem> reflectiveClasses() {
-        return Arrays.asList(ReflectiveClassBuildItem.builder("com.splunk.Index").build(),
-                ReflectiveClassBuildItem.builder("com.splunk.SavedSearch").build(),
-                ReflectiveClassBuildItem.builder("com.splunk.Service").build());
+        return Arrays.asList(ReflectiveClassBuildItem.builder("com.splunk.Index").constructors().build(),
```

Comment:
If these are public classes, it's better to use `Class.getName()` instead of hard coding it.